crates/proof-of-sql/src/proof_primitive/dory/dynamic_dory_helper.rs

```diff
@@ -42,6 +51,37 @@ pub(super) fn compute_dynamic_T_vec_prime(
     T_vec_prime
 }
 
+/// Compute the commitments to the rows of the matrix M that is derived from `a`.
+#[cfg(feature = "blitzar")]
+pub(super) fn compute_dynamic_T_vec_prime(
+    a: &[F],
+    nu: usize,
+    prover_setup: &ProverSetup,
+) -> Vec<G1Affine> {
+    let (blitzar_output_bit_table, blitzar_output_length_table, blitzar_scalars) =
+        create_blitzar_metadata_tables_for_T_vec_prime_setup(a);
```

Comment:
Using the existing conversion from a Scalar slice along with `bytemuck::TransparentWrapper` gives you this for free.
```suggestion
    let a_col =
        CommittableColumn::from(TransparentWrapper::wrap_slice(a) as &[DoryScalar]);
    let (blitzar_output_bit_table, blitzar_output_length_table, blitzar_scalars) =
        create_blitzar_metadata_tables(&[a_col], 0);
```

Reply:
This is very nice, thank you!